src/canadiantracker/http.py

```diff
@@ -13,19 +14,34 @@
 _repository = canadiantracker.storage.get_product_repository_from_sqlite_file(_db_path)
 
 
+# This request takes a few seconds to execute, causing some delay when
+# accessing the products page.  Since the contents of the database never
+# changes while we're running, cache it indefinitely (use an arbitrarily
+# long expiration time, since there doesn't seem to be a way to say
```

Comment:
Not sure what you mean, or maybe this comment about "long expiration time" is outdated?

Reply:
You are right, this is stale, from when I was trying to use some fastapi caching feature.